ui/helpers/utils/i18n-helper.tsx

```diff
@@ -110,19 +115,21 @@ export async function fetchLocale(localeCode) {
 
 const relativeTimeFormatLocaleData = new Set();
 
-export async function loadRelativeTimeFormatLocaleData(localeCode) {
+export async function loadRelativeTimeFormatLocaleData(
+  localeCode: string,
+): Promise<any> {
   const languageTag = localeCode.split('_')[0];
   if (
     Intl.RelativeTimeFormat &&
-    typeof Intl.RelativeTimeFormat.__addLocaleData === 'function' &&
+    typeof (Intl.RelativeTimeFormat as any).__addLocaleData === 'function' &&
     !relativeTimeFormatLocaleData.has(languageTag)
   ) {
     const localeData = await fetchRelativeTimeFormatData(languageTag);
-    Intl.RelativeTimeFormat.__addLocaleData(localeData);
+    (Intl.RelativeTimeFormat as any).__addLocaleData(localeData);
```

Comment:
I searched for details on this __addLocaleData method... It seems it comes from @formatjs, but I think Intl is a global object on window now. @Gudahtt might know more, but for now casting this to any is fine unless @Gudahtt knows what is going on here. I tried using a declaration block to add a type signature to this method to no avail.

Reply:
I did some research on this and it seemed to come from windows. (same as a globe object i was dealing with);

>  I tried using a declaration block to add a type signature to this method to no avail.
Can you show me your another solution?